_pages/events/screenings.md

```diff
@@ -7,6 +7,40 @@
 
 MITOC and [MIT LSC](http://lsc.mit.edu/) are pleased to bring you
 
+### No Man’s Land Film Festival
+
+on **Thursday, February 27 @ 7:00 PM** in [MIT Building 26-100](http://whereis.mit.edu/?go=26)
+
+**_$5 with a valid MIT ID or $10 for General Admission_**
+
+**Purchase your ticket today online!**
+
+Click [here](/pay) or go to the Pay MITOC link above
```

Comment:
Generally best to [avoid saying click here](https://www.w3.org/QA/Tips/noClickHere) (for accessibility reasons). Instead, maybe just `[Pay MITOC](/pay) and under Trip and Event Fees,...`